<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubjectAdding</class>
 <widget class="QWidget" name="SubjectAdding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(65, 115, 176)</string>
  </property>
  <widget class="QWidget" name="widget_AddingSubject" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>411</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(219, 226, 239);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color: rgb(18, 46, 79);</string>
   </property>
   <widget class="QPushButton" name="pushButton_Add">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>490</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(18, 46, 79);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color: rgb(18, 46, 79);
font: 75 20pt &quot;Gentium Book Basic&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Add </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Teacher">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color: rgb(18, 46, 79);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Teachers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Red">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Yellow">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color:rgb(255, 255, 0)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Green">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(0, 170, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Blue">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color:rgb(85, 0, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pink">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color:rgb(255, 85, 255)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_White">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Orange">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(255, 170, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LightGreen">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(85, 255, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LightBlue">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(85, 170, 255)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Purple">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(170, 85, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Brown">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(203, 102, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pushButton_Add</zorder>
   <zorder>listWidget_Teachers</zorder>
   <zorder>pushButton_Red</zorder>
   <zorder>pushButton_Yellow</zorder>
   <zorder>pushButton_Green</zorder>
   <zorder>pushButton_Blue</zorder>
   <zorder>pushButton_Pink</zorder>
   <zorder>pushButton_White</zorder>
   <zorder>pushButton_Orange</zorder>
   <zorder>pushButton_LightGreen</zorder>
   <zorder>pushButton_LightBlue</zorder>
   <zorder>pushButton_Purple</zorder>
   <zorder>pushButton_Brown</zorder>
   <zorder>lineEdit_Teacher</zorder>
  </widget>
  <widget class="QLabel" name="label_AddingSubject">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);</string>
   </property>
   <property name="text">
    <string>Adding subject</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
border-radius:15px;
border-style: solid;
border-width: 1px;
border-color: rgb(18, 46, 79);
font: 14pt &quot;Gentium Book Basic&quot;;
color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Teacher">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);</string>
   </property>
   <property name="text">
    <string>Teacher:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Color">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Gentium Book Basic&quot;;
color:black;
background-color:rgb(219, 226, 239);</string>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
